api/src/main/java/eu/darkbot/api/managers/InventoryAPI.java

```diff
@@ -0,0 +1,32 @@
+package eu.darkbot.api.managers;
+
+import java.util.List;
+
+/**
+ * Provides Inventory Data
+ */
+
+public interface InventoryAPI {
+    /**
+     * @return list of items in inventory
+     */
+    List<? extends Item> getItems();
```

Comment:
```suggestion
    List<? extends Item> getItems(int minWaitMs);
```

Adding a "min wait" allows the bot to not need to re-fetch them every tick.
If a plugin asks for `getItems(1000)` it means it's "ok" if the results are up to 1s old
If a plugin asks for `getItems(0)` it means they need results from this very tick